packages/tools/pie-css/scss/mixins/visuallyHidden.scss

```diff
@@ -0,0 +1,10 @@
+// Mixin to hide an element yet keep it available for screen readers.
+@mixin visually-hidden() {
```

Comment:
I'm unsure about adding this here, however if we are then it needs to be integrated into the css validity test as outlined in the package readme. Please take a look at `validityTest.scss`

Reply:
done